utils/ios.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+type Document struct {
+	File         *os.File
+	FileLocation string
+}
+
+func NewIosDocument(lang string) *Document {
+	filePath := fmt.Sprintf("./output/iOS/%s.strings", lang)
+	file, err := os.OpenFile(filePath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		log.Fatal("Error opening file:", err)
+	}
+
+	return &Document{
+		File:         file,
+		FileLocation: fmt.Sprintf("./output/iOS/%s.strings", lang),
```

Comment:
we can use a variable from line 15 krub.

Reply:
Thanks @icecreamhotz, Cleaning up...